backend/python/app/services/implementations/story_service.py

```diff
@@ -262,23 +259,30 @@ def get_story_translations(
                 .order_by(Story.id)
                 .all()
             )
+
             translator = aliased(User)
             reviewer = aliased(User)
 
             story_translation_data = (
-                db.session.query(StoryTranslation, translator, reviewer)
-                .join(Story, Story.id == StoryTranslation.story_id)
+                db.session.query(
+                    StoryTranslationAll,
```

Comment:
changed to `StoryTranslationAll` to handle joining with story translation contents